index.bs

```diff
@@ -205,6 +208,11 @@ partial interface WorkerGlobalScope {
 };
 </pre>
 
+<p class="note">
+It is important that `console` is always visible and usable to scripts,
```

Comment:
I'm not sure backticks will work here; you might need to use `<code>`. Check the output.